catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/impl/MockSource.java

```diff
@@ -109,7 +111,11 @@ public String getVersion() {
 
     @Override
     public SourceResponse query(QueryRequest query) {
-        return null;
+        return sourceResponse;
+    }
+
+    public void setQueryResponse(SourceResponse sourceResponse) {
```

Comment:
I will take another look and try to find a workaround again. I remember this being the only option for `MockSource` - Perhaps it would be better to use a different class, but it seems to me that `MockSource` should be used here for mocking a source :).